tests/functional/parsing/test_parsing_hunt.py

```diff
@@ -27,7 +27,7 @@ def test_hunt_command_full_parsing(database, monkeypatch):
     args_list = ["hunt", "-n", "test",
                  "--config", "./orion_config_random.yaml",
                  "--max-trials", "400", "--pool-size", "4",
-                 "./black_box.py", "-x~normal(1,1)"]
+                 "./black_box.py", "-x~normal(1,1)", "--worker-trials 5"]
```

Comment:
As it is, the option ```--worker-trials``` is an argument to the user's script. It should be moved along with ```--max-trials``` and ```--pool-size```